scipy/stats/_continuous_distns.py

```diff
@@ -8703,6 +8703,15 @@ def _logsf(self, x, a, b):
             logsf[i] = np.log1p(-np.exp(self._logcdf(x[i], a[i], b[i])))
         return logsf
 
+    def _entropy(self, a, b):
+        A = sc.ndtr(a)
+        B = sc.ndtr(b)
```

Comment:
```suggestion
        A = _norm_cdf(a)
        B = _norm_cdf(b)
```

Your code is not wrong but throughout this file `_norm_cdf` is used for better readability instead of the cryptic acronym from `scipy.special`.